### DISCUSSION TITLE
Dark mode for docs

### DISCUSSION THREAD

**Discussion by prathu9** [pmndrs/zustand#2216] [Ideas] [answered]
Currently in docs there is no dark mode. It is very difficult to read in bright white light.
Please add dark mode in docs,it will be very helpful.

**Comment by dai-shi** [comment, marked as answer]
The website is hosted under a different repo: https://github.com/pmndrs/website/tree/docs
Would you like to contribute?
cc: @CodyJasonBennett

**Comment by prathu9** [comment]
I will check it.

**Comment by Ali-Hussein-dev** [comment]
if @prathu9 can't work, I would like to contribute @dai-shi

**Reply by prathu9** [reply]
You can do it.
